src/plugins/charts/public/services/colors/mapped_colors.ts

```diff
@@ -18,14 +18,14 @@
  */
 
 import _ from 'lodash';
-import d3 from 'd3';
+import Color from 'color';
 
 import { CoreSetup } from 'kibana/public';
 
 import { COLOR_MAPPING_SETTING } from '../../../common';
 import { createColorPalette } from './color_palette';
 
-const standardizeColor = (color: string) => d3.rgb(color).toString();
+const standardizeColor = (color: string) => new Color(color).hex().toLowerCase();
```

Comment:
:information_source: The old code used d3.toString() which created a lower-case hex value, to make sure I don't need to touch the tests and we can thus better guarantee that this doesn't change any functionality, I kept here the same logic, even though technically we could use `.hex()` as well and just adjust the tests.